prog/weave-kube/weave-monitoring-svc.yaml

```diff
@@ -0,0 +1,28 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+   name: weave-net-monitoring
+   namespace: kube-system
+   annotations:
+     prometheus.io.scrape: "true"
```

Comment:
The standard config we have scrapes all services, so this probably won't be needed.